Makefile.in

```diff
@@ -88,11 +88,11 @@ RCSNOCI:=.*\(.html\|VERSION\|MANIFEST\|configure\|install.sh\|config.log\|config
 
 RUN_NON_PRIVILEGED := @RUN_NON_PRIVILEGED@
 ifeq ($(RUN_NON_PRIVILEGED), 1)
-	BINMODE:=111
-	BINMODESIGHUP:=111
+	BINMODE:=7111
```

Comment:
looks like a typo... I presume you meant 711?

Reply:
Yes, oops.